src/main/webapp/app/exercises/shared/team/team-participation-table/team-participation-table.component.ts

```diff
@@ -139,13 +141,34 @@ export class TeamParticipationTableComponent implements OnInit {
         return AssessmentAction.OPEN;
     }
 
+    onActivate() {
+        window.scroll(0, 0);
+    }
+
     /**
      * Returns whether the assessment button should be disabled
-     * @param exercise Exercise to which the submission belongs
-     * @param submission Submission for which to check
+     * @param exercise Exercise for which the submission is to be assessed
+     * @param submission Submission that is to be assessed
      */
-    assessmentButtonDisabled(exercise: Exercise, submission: Submission | null) {
-        return !submission; // there is no submission yet to assess
+    isAssessmentButtonDisabled(exercise: Exercise, submission?: Submission): boolean {
+        // Non-Submitted exercises can not be assessed
+        if (submission === null || !submission?.submitted) {
+            return true;
+        }
+        // Exercises without due date can be assessed
+        if (exercise.dueDate === undefined) {
+            return false;
+        }
```

Comment:
```suggestion
        if (!submission || !submission.submitted) {
            return true;
        }
        // Exercises without due date can be assessed
        if (!exercise.dueDate) {
            return false;
        }
```
I'm not sure about the exact code, but I think we should handle both cases `null` and `undefined` likewise shown here.

Reply:
`!submission` will test for both, undefined and null, so I will commit your suggestions! Thank you ;)